### PR TITLE
feat: Clarify custom peer disconnection

### PR DESCRIPTION
**Pull request by JeremiGendron**
## Proposed changes

 - Implement v-if/else-if/else switch for peer/custom/disconnecting
 - Provide "Disconnect from peer" instead of "Connect to best" when connected to custom peer
 - Automatically refresh (and show through refresh button rotation) when user disconnects from peer
 - Use the cross svg with matching styling to the refresh button
 - hover:bg-theme-option-button-hover hover:text-grey-light

Looks and feels very nice. Planning to add more functionality to the custom peer interface as mentionned in the Ref below. I think we should provide a timeout window for reconnecting when a custom peer disconnects which the user can manually terminate (alongside the presented disconnect button).

Fixes: #938
Ref: #981

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (improve a current implementation without adding a new feature or fixing a bug)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build (changes that affect the build system)
- [ ] Docs (documentation only changes)
- [ ] Test (adding missing tests or fixing existing tests)
- [ ] Other... Please describe:

## Checklist

- [x] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

image

image

After click
image

After disconnect
image

### CODE REVIEW

**Comment by alexbarnsley** [comment]
I think it looks good. My only tweak would be to add a hover-over to the reload button. Possibly to match the reload button hover on the import page, but the dark mode colour 🤔

**Comment by JeremiGendron** [comment]
alright, I'll do that now @alexbarnsley

**Comment by alexbarnsley** [comment]
thanks @JeremiGendron 👌

**Comment by JeremiGendron** [comment]
And light theme
image

**Comment by ItsANameToo** [comment]
@JeremiGendron just a note, it's better if you leave out the stuff between brackets after the commit keywords like `feat` and `refactor`, since they don't add any value in this repo. It's used in core to denote the package that is worked on, but here we don't have different packages

**Comment by JeremiGendron** [comment]
@ItsANameToo got it, thanks!

File: src/renderer/components/App/AppSidemenu/AppSidemenuNetworkStatus.vue [hunk 1]
```diff
@@ -196,6 +224,7 @@ export default {
 
   methods: {
     async refreshPeer () {
+      console.log('clicked')
```
Comment (alexbarnsley) [inline src/renderer/components/App/AppSidemenu/AppSidemenuNetworkStatus.vue]: debug

**Comment by alexbarnsley** [comment]
Just a debug line that needs removing then can be merged 👌

**Pull request merged**